test/e2e/rollout_test.go

```diff
@@ -3663,6 +3663,96 @@ var _ = SIGDescribe("Rollout", func() {
 			CheckPodBatchLabel(workload.Namespace, workload.Spec.Selector, "2", "4", 1)
 			CheckPodBatchLabel(workload.Namespace, workload.Spec.Selector, "2", "5", 1)
 		})
+
+		It("patch batch id to pods: only rollout-id changes", func() {
```

Comment:
add a case for rolling to 20%, and rollout change rollout id

Reply:
I rolling 60%, and change rollout-id in `Line 3731`.

Reply:
done